spec/openbd/resources/hanmoto_spec.rb

```diff
@@ -0,0 +1,129 @@
+    EOB
+  end
+
+  let(:hanmoto_item) do
+    json = JSON.parse(sample_hanmoto_data)
+    OpenBD::Resources::Hanmoto.new(json)
+  end
+
+  let(:hanmoto_item2) do
+    json = JSON.parse(sample_hanmoto_data2)
+    OpenBD::Resources::Hanmoto.new(json)
+  end
+
+  let(:hanmoto_item3) do
+    json = JSON.parse(sample_hanmoto_data3)
+    OpenBD::Resources::Hanmoto.new(json)
+  end
+
+  it "has a dateshuppan" do
+    expect(hanmoto_item.dateshuppan).to be == "2016-06"
+  end
+
+  it "has a ruishokyougousho" do
+    expect(hanmoto_item.ruishokyougousho).to be == "広井良典『人口減少社会という希望』朝日選書，加藤久和『世代間格差：人口減少社会を問いなおす 』ちくま新書など，人口問題に関する書籍。心理学的な観点に立った書籍は本邦初。"
+  end
+
+  it "has jyuhan" do
+    expect(hanmoto_item2.jyuhan[0].class).to be == OpenBD::Resources::Hanmoto::Jyuhan
+    expect(hanmoto_item2.jyuhan[0].date).to be == "2016-06-23"
+    expect(hanmoto_item2.jyuhan[0].comment).to be == "とくに石垣が美しいことで有名な熊本城。本書はその姿をあえてモノクロームで表現しました。2016年4月の熊本地震で大きな被害をうけましたが、当時のうつくしく力強い姿をこれからに残すため、今回緊急増刷することにしました。"
+    expect(hanmoto_item2.jyuhan[0].ctime).to be == "2016-06-09 13:43:09"
+    expect(hanmoto_item2.jyuhan[0].suri).to be == 3
+  end
+
+  it "has an obinaiyou" do
+    expect(hanmoto_item.obinaiyou).to be == "人口が減り始めた日本。\n私たちは命にどう関わるべきか？\n命についての問題―生殖補助医療，育児不安，母性，親子，介護，人生の終末―に直面し苦悩し，格闘する心を扱う「人口の心理学」の提案！\n心理学のみならず，人口学，社会学，生命倫理，日本近代史の第一線の論客が結集し，少子化，高齢化，人口減少に直面する日本社会のあり方を問う。"
+  end
+
+  it "has a bikoutrc" do
+    expect(hanmoto_item.bikoutrc).to be == "トランスビュー取り扱いです"
+  end
+
+  it "doesnt have a zasshicode" do
+    expect(hanmoto_item.zasshicode).to be_nil
+  end
+
+  it "doesnt have a genrecodetrcjidou" do
+    expect(hanmoto_item.genrecodetrcjidou).to be_nil
+  end
+
+  it "has a bessoushiryou" do
+    expect(hanmoto_item.bessoushiryou).to be == "序章　人口の心理学の視点―命と死と生涯発達　●柏木惠子\n第Ⅰ部　誕生―「授かる命」から「つくる命」へ\n第1章　生殖補助医療・不妊治療のいま―心とテクノロジー　●小泉智恵・平山史朗\n第2章　産む選択、産まない選択―出生前診断　●玉井真理子\n第3章　近代日本社会と子どもの命―子返しの習俗と規範の形成　●太田素子\n第4章　血がつながらない子どもの親になる―特別養子縁組による親子の形　●富田庸子\n第Ⅱ部　親子関係―「少子の子ども」と「長命の親」\n第5章　子どもの価値―なぜ、女性は子どもを産むのか　●永久ひさ子\n第6章　育児不安を考える―ライフコースの激変とアイデンティティの揺らぎ　●柏木惠子・加藤邦子\n第7章　もたれ合う家族―日本の家族文化の問題　●舩橋惠子\n第8章　家族が変わる、老親介護も変わる―二一世紀の高齢者の介護と暮らし　●染谷俶子\n第Ⅲ部　命の終わり方―「長命」は「長寿」か\n第9章　人間の尊厳と死―「死の尊厳」の語られ方を読み解く　●大谷いづみ\n第10章　変わるお葬式、消えるお墓―その実態と現代人の意識　●小谷みどり\n第11章　長生きすること―長命の価値と課題　●森岡清美\n終章　少子高齢社会の命と心―現在とこれから　●高橋惠子\n\nコラム \n1　子どもの誕生と死の意味―先人の日記や手紙からの示唆　●柏木惠子\n2　将来の日本がもつ人口問題とは？　●別府志海\n3　日本の貧困　●阿部彩\n4　性・生殖と政治　●高橋惠子\n5　誕生のインファンティア　●西平直\n6　生殖補助医療の死角―当事者の視点から　●加藤英明\n7　「マタニティ・ハラスメント」は女性の身体性への差別　●杉浦浩美\n8　結婚、出産の価値の変化　●本田由紀\n9　なぜ少子に虐待か―家族臨床から見えること　●平木典子\n10　嬰児殺に見る命の重み　●川﨑二三彦\n11　「親孝行の終焉」の示唆するもの　●深谷昌志\n12　平均寿命と健康寿命　●菅原育子\n13　介護保険制度　●神前裕子\n14　江戸時代の高齢化と看取りのシステム　●柳谷慶子\n15　老人ホームに住むという選択　●神前裕子\n16　親孝行は美徳か？―親子間の資源の流れ再考　●柏木惠子\n17　長命化で厳しさを増す親と子のライフプラン　●畠中雅子"
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [1019/200] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->